mempool/src/counters.rs

```diff
@@ -429,6 +430,20 @@ pub fn shared_mempool_pending_broadcasts(peer: &PeerNetworkId) -> IntGauge {
     ])
 }
 
+/// Counter tracking the number of peers that changed priority in shared mempool
+static SHARED_MEMPOOL_PRIORITY_CHANGE_COUNT: Lazy<IntGaugeVec> = Lazy::new(|| {
```

Comment:
Aah, thanks @bchocho -- I missed `IntGauge`  🤦  Will change to that. Note: I'm not sure `Histogram` makes sense because we update this metric really infrequently. It's probably easiest to just see the raw gauge value instead of try to work with averages, rate changes, etc.